README.md

```diff
@@ -4,13 +4,45 @@ This repository contains samples to experiment with rendering into multiple view
 aspirational and may not actually work. Some samples may require a custom engine build to run. Some samples may be
 outdated and/or have other issues. DO NOT DEPEND ON ANYTHING IN THIS REPOSITORY.
 
-To run a sample, execute:
+## How to use?
 
+This sample must be used with the correct revision of a custom engine.
+Currently, it only supports macOS.
+
+## Setup
```

Comment:
nit: are these supposed to be subheadings under "How to use?" - if so, they need three #

```suggestion
### Setup

```

And also should be followed by a blank line.